src/components/ArcadeMode.jsx

```
@@ -0,0 +1,548 @@
+          gameMode="arcade"
+          onGameComplete={handleGameComplete}
+          onBackToMenu={onBackToMenu}
+          selectedCharacter={selectedCharacter}
+          currentAI={currentAI}
+          tournamentManager={tournamentManager}
+        />
+      ) : null
+
+    case 'endingCinematic':
+      const isVictorious = tournamentManager?.isVictorious || false
+      return (
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)